CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
 * Accessibility
   * VoiceOver improvements
   * Fix color contrast ratios
+* Increase hit area for several buttons
```

Comment:
Whoops it looks like this line got duplicated. It should be under the "Accessibility" section.
```suggestion
  * Increase hit area for several buttons
```